### DISCUSSION TITLE
ScriptReference/Rigidbody2D.ClosestPoint

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#638] [Notes]
# ScriptReference/Rigidbody2D.ClosestPoint

https://docs.unity3d.com/ScriptReference/Rigidbody2D.ClosestPoint.html

**Comment by qwe321qwe321qwe321** [comment]
[Crashes in Rigidbody2D.ClosestPoint() with inactive gameObject(rigidbody2D).](https://forum.unity.com/threads/crashes-in-rigidbody2d-closest2d-vector2-with-inactive-gameobject-rigidbody2d.1088902/) (Fixed in 2021.1)
https://issuetracker.unity3d.com/issues/crash-when-calling-rigidbody2d-dot-closestpoint-on-an-inactive-rigidbody2d
